.circleci/config.yml

```diff
@@ -60,12 +60,19 @@ jobs:
       - run:
           name: e2e test
           command: |
+            set +e
             if [[ $CIRCLE_BRANCH != *"master"* || ! -z $CIRCLE_TAG ]] ; then
-              E2EPROFILE=saucelight npm run test:e2e
+              E2EPROFILE=saucelight npm run test:e2e; STATUS=$?;
             else
-              npm run test:e2e
+              npm run test:e2e; STATUS=$?;
```

Comment:
wouldnt be more readable if status is moved after if?

Reply:
I do not know why, but I like one-liners in bash. But if you say it is more readable, I can do that.